core/vm/contracts.go

```diff
@@ -506,6 +522,10 @@ func (c *transfer) RequiredGas(input []byte) uint64 {
 }
 
 func (c *transfer) Run(input []byte, caller common.Address, evm *EVM, gas uint64) ([]byte, uint64, error) {
+	gas, err := debitRequiredGas(c, input, gas)
```

Comment:
I don't think we actually want to debit gas here. Or, we want to return something much smaller than `TxGas` above. Maybe it should be `CallValueTransferGas`? Wdyt?

Reply:
ok, didn't check what `TxGas` is.